tidb-cloud/tidb-cloud-billing.md

```diff
@@ -80,6 +80,24 @@ The billing details page shows the billing summary by project and by service. Yo
 > - The total amount in the monthly bill is rounded off to the 2nd decimal place.
 > - The total amount in the daily usage details is accurate to the 6th decimal place.
 
```

Comment:
https://docs.pingcap.com/tidbcloud/manage-user-access#organization-role

Do we also need to add the info about viewing usage costs here?

![image](https://github.com/pingcap/docs/assets/79440533/0e7e0143-264e-4516-ae27-fbccbafb5b0b)

Reply:
https://github.com/pingcap/docs/pull/16608/commits/2f0b028936c729833deada833e7e874bc464d962